src/helpers/figures/charts/scorecard_chart_config_builder.ts

```diff
@@ -133,14 +136,28 @@ class ScorecardChartConfigBuilder {
       style.baselineDescr.font
     ).width;
 
+    let baseline_x: number;
+
+    switch (this.runtime.baselineStyle?.align) {
+      case "right":
+        baseline_x = this.width - CHART_PADDING - baselineDescrWidth - baselineWidth;
+        break;
+      case "left":
+        baseline_x = CHART_PADDING + baselineArrowSize;
+        break;
+      case "center":
+      default:
```

Comment:
```suggestion
      default:
```
or add all cases explicitly and remove `default`